html/controls/operator.js

```diff
@@ -1435,6 +1435,8 @@ function createTeamsContent() {
 		var teamid = _crgUtils.checkSbId(teamname);
 		$sb("Teams.Team("+teamid+").Name").$sbSet(teamname);
 		newTeamName.val("").keyup().focus();
+		// Presume we'll have gotten the XML update within 100ms.
+		setTimeout(function(){selectTeam.val(teamname);}, 100);
```

Comment:
How certain are you that 100ms is sufficient, especially when the operator screen is not on the same machine as the backend?

Reply:
It'll all be in the same sports hall, and thus within 1-2ms. I'd be more worried about the backend being slow.

Reply:
This only selects the team in the dropdown but doesn't change the bottom part.